embedder-tests/savant/src/test/resources/logback-test.xml

```diff
@@ -26,9 +26,9 @@
   </appender>
   
   <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-    <file>${indy.home}/var/log/indy/indy.log</file>
+    <file>target/var/log/indy/indy.log</file>
```

Comment:
This will effectively cause two concurrent functional tests to write logs to the same file during the build. Can you use the AbstractIndyFunctionalTest to retrieve the options and get the value of `${indy.home}` from there, if you need to scan through the logs?